### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_coverlet.msbuild_Used By-NuGet packages_DeveloperTool : Table name is not set for the 'NuGet packages' table present on the screen.

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9042]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468937

### Repro Steps

1. Launch the application URL: https://www.nuget.org/packages/coverlet.msbuild/
2. "coverlet.msbuild" page will get open.
3. Tab to 'Used by' tab controls and press enter to select it.
4. Tab to 'NuGet packages' table.
5. Open developer tool by using ctrl+shift+c and hover over 'NuGet packages' table to check its name. 

**Actual Behavior**:

Table name is not set for the 'NuGet packages' table present on the screen.

Similar issue is observed in through out the application where ever such tables are present.

### Expected Behavior

Proper name should be set for the 'NuGet packages' table present on the screen as per the data/purpose of the table
